crates/alkahest-renderer/src/ecs/utility.rs

```diff
@@ -184,55 +307,103 @@ impl Utility for Route {
     }
 }
 
+impl Utility for RouteNode {
+    fn icon() -> Icon {
+        Icon::Unicode(ICON_MAP_MARKER)
+    }
+
+    fn label(str: &str) -> Label {
+        Label::from(str).with_offset(0.0, 0.0, 0.12)
+    }
+
+    fn default_label() -> Label {
+        Label::new_default("").with_offset(0.0, 0.0, 0.12)
+    }
+}
+
+impl RouteNode {
+    pub fn make_budle(parent: Entity, node: RouteNodeHolder) -> RouteNodeBundle {
```

Comment:
aside from the typo, i think it makes semantically more sense to implement this on the RouteNodeBundle type itself
```suggestion
impl RouteNodeBundle {
    pub fn new(parent: Entity, node: RouteNodeHolder) -> RouteNodeBundle {
```